Requirement on dask

Hi @pnsaevik , start it!

My first try with `pytest` failed due to missing dask, which is not currently an explicit dependency. I didn't investigate the details, but it ran fine once I manually installed it in my testing environment. You might need to change your code for an alternative procedure when missing it or add it as an explicit dependency.

```
FAILED tests/test_io.py::Test_Ambient_from_config::test_roms_file - ValueError: unrecognized chunk manager dask - must be one of: []
FAILED tests/test_roms.py::Test_open_dataset::test_correct_dims_when_multi_file - ValueError: unrecognized chunk manager dask - must be one of: []
FAILED tests/test_roms.py::Test_open_location::test_correct_vars_and_dims - ValueError: unrecognized chunk manager dask - must be one of: []
3 failed, 52 passed, 34 warnings in 26.59s
```

Somehow related to this, it would be probably a good idea to set up a protection rule in GitHub to run your tests when pushing into the `main`. If interested, I can guide you through how to do that.

--- a/.circleci/config.yml
+++ b/.circleci/config.yml
@@ -18,7 +18,7 @@ jobs:
       - checkout
       - run:
           name: "Install prerequisites"
-          command: pip install pytest sphinx dask matplotlib
+          command: pip install pytest sphinx matplotlib
       - run:
           name: "Install package"
           command: pip install .

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -5,6 +5,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased] - 2023-06-07
+### Fixed
+- Included dask as an explicit dependency
+
 ## [1.2] - 2023-04-14
 ### Added
 - Ambient input from ROMS ocean model

--- a/setup.cfg
+++ b/setup.cfg
@@ -31,6 +31,7 @@ install_requires =
     netcdf4
     holoviews
     scipy
+    dask
     tomli
 python_requires = >=3.6
 

